src/docs/product/issues/index.mdx

```diff
@@ -32,6 +32,26 @@ From the **Issues** page, you can begin to triage. The page is organized into ta
 
 Learn more about triaging issues and their different states in [Issue States and Triage](/product/issues/states-triage/).
 
+### Display Options
+
+You can use the display dropdown in the issues page to see error counts as a % of [sessions](/product/releases/health/#session).
```

Comment:
Because this section is called "Display Options" not "Events as %", we should probably at least mention both of display options. Suggest:
 
```suggestion
You can use the display dropdown on the **Issues** page to see errors as a count of events or as a percentage of [sessions](/product/releases/health/#session).
```

Reply:
Yes, looks better.